doc/papers/joss/paper.bib

```diff
@@ -0,0 +1,383 @@
+                  Dudouit, Yohann and
+                  Karakus, Ali and
+                  Kerkemeier, Stefan and
+                  Lan, YuHsiang and
+                  Medina, David and
+                  Merzari, Elia and
+                  Min, Misun and
+                  Parker, Scott and
+                  Ratnayaka, Thilina and
+                  Smith, Cameron and
+                  Sprague, Michael and
+                  Stitt, Thomas and
+                  Thompson, Jeremy and
+                  Tomboulides, Ananias and
+                  Tomov, Stanimire and
+                  Tomov, Vladimir and
+                  Vargas, Arturo and
+                  Warburton, Tim and
+                  Weiss, Kenneth},
+  title        = {{CEED ECP Milestone Report: Improve performance and
+                   capabilities of CEED-enabled ECP applications on
+                   Summit/Sierra}},
+  year         = 2020,
+  publisher    = {Zenodo},
+  doi          = {10.5281/zenodo.3860804}
+}
+
+@misc{CEEDwebsite,
+  title = "CEED",
+  url = "https://ceed.exascaleproject.org/",
+  year = "2020"
+}
+
+@misc{CUDAwebsite,
+  title = "CUDA",
+  url = "https://developer.nvidia.com/about-cuda",
+  year = "2020"
+}
+
+@article{Fischer2020scalability,
+  title={Scalability of high-performance PDE solvers},
+  author={Paul Fischer and Misun Min and Thilina Rathnayake and Som Dutta and Tzanio Kolev and Veselin Dobrev and Jean-Sylvain Camier and Martin Kronbichler and Tim Warburton and Kasia Świrydowicz and Jed Brown},
+  journal={The International Journal of High Performance Computing Applications},
+  year={2020},
+  doi = "10.1177/1094342020915762"
+}
+
+@book{Fortran77-lang,
+  title={Standard X3. 9-1978, Programming Language Fortran (revision of ANSI X2. 9-1966)},
+  author={ANSI},
+  journal={New York: ANSI},
+  year={1978}
+}
+
+@misc{HIPwebsite,
+  title = "HIP",
+  url = "https://rocmdocs.amd.com/en/latest/Programming_Guides/HIP-GUIDE.html",
+  year = "2020"
+}
+
+@article{Julia-lang,
+  title={Julia: A fresh approach to numerical computing},
+  author={Bezanson, Jeff and Edelman, Alan and Karpinski, Stefan and Shah, Viral B},
+  journal={SIAM {R}eview},
+  volume={59},
+  number={1},
+  pages={65--98},
+  year={2017},
+  publisher={SIAM},
+  doi={10.1137/141000671}
+}
+
+@misc{libceed-user-manual,
+  author       = {Abdelfattah, Ahmad and
+                  Barra, Valeria and
+                  Beams, Natalie and
+                  Brown, Jed and
+                  Camier, Jean-Sylvain and
+                  Dobrev, Veselin and
+                  Jean-Sylvain, Yohann and
+                  Ghaffari, Leila and
+                  Kolev, Tzanio and
+                  Medina, David and
+                  Rathnayake, Thilina and
+                  Thompson, Jeremy L and
+                  Tomov, Stanimire},
+  title        = {libCEED User Manual},
+  month        = sep,
+  year         = 2020,
+  publisher    = {Zenodo},
+  version      = {0.7},
```

Comment:
I should have put my comment on this line. Perhaps it's clearer what I wanted to say. 
I meant to say that the manual on Zenodo should be updated to (at least) 0.8. Then we can update its citation here.